primefaces/src/main/java/org/primefaces/renderkit/HeadRenderer.java

```diff
@@ -160,6 +151,17 @@ public void encodeBegin(FacesContext context, UIComponent component) throws IOEx
             }
         }
 
+        //Registered Resources
+        UIViewRoot viewRoot = context.getViewRoot();
+        List<UIComponent> resources = new ArrayList<>(viewRoot.getComponentResources(context, "head"));
+        moveResourceToTop(resources, "primeicons/primeicons.css");
+        moveResourceToTop(resources, "theme.css");
+        for (int i = 0; i < resources.size(); i++) {
+            UIComponent resource = resources.get(i);
+            LOGGER.log(Level.FINE, "HeadRenderer resource: " + resource.getAttributes().get("name"));
```

Comment:
user lambda or remove, currently the string is always concatinated

Reply:
Changed to Lambda